jmix-reports/reports-flowui/src/main/java/io/jmix/reportsflowui/view/run/ReportTableView.java

```diff
@@ -124,53 +137,56 @@ public void setReportParameters(Map<String, Object> reportParameters) {
         this.reportParameters = reportParameters;
     }
 
+    /**
+     * @deprecated use {@link #setDocument(ReportOutputDocument)}
+     */
+    @Deprecated(since = "2.1.3", forRemoval = true)
     public void setTableData(byte[] tableData) {
         this.tableData = tableData;
     }
 
+    public void setDocument(ReportOutputDocument document) {
```

Comment:
Maybe call this method `setReportOutputDocument` to make things more clear? "Document" sounds to generic. WDYT?